src/common/services/api/order.service.js

```diff
@@ -179,6 +181,21 @@ class Order{
     });
   }
 
+  checkErrors(){
+    return this.cortexApiService.get({
+      path: ['carts', this.cortexApiService.scope, 'default'],
+      zoom: {
+        needInfo: 'order:needinfo'
+      }
+    })
+      .map((data) => {
+        return map(data.needInfo, 'name');
+      })
+      .do((errors) => {
+        this.$log.error('The user was presented with these `needinfo` errors. They should have been caught earlier in the checkout process.', errors);
```

Comment:
My thought was to pipe `$log` to rollbar eventually but I've never actually done that. Do you guys have any thoughts?